github.js

```diff
@@ -199,20 +148,14 @@ function configureCredentials(config, ui) {
 
   return Promise.resolve()
   .then(function() {
-    ui.log('info', 'If using two-factor authentication or to avoid using your password you can generate an access token at %https://' + (config.hostname || 'github.com') + '/settings/tokens%. Ensure it has `public_repo` scope access.');
-    return ui.input('Enter your GitHub username');
+    ui.log('info', 'You can generate an access token at %https://' + (config.hostname || 'github.com') + '/settings/tokens%.');
+    return ui.input('Enter your GitHub access token');
   })
-  .then(function(username) {
-    auth.username = username;
-    if (auth.username)
-      return ui.input('Enter your GitHub password or access token', null, true);
-  })
-  .then(function(password) {
-    auth.password = password;
-    if (!auth.username)
-      return false;
-
-    return ui.confirm('Would you like to test these credentials?', true);
```

Comment:
Backwards compat should really mean the same prompts too I think.
On Sun, 12 Jun 2016 at 02:49, Tamir Duberstein <notifications@github.com>
wrote:

> In github.js <https://github.com/jspm/github/pull/90#discussion_r66714112>
> :
>
> >    })
> > -  .then(function(username) {
> > -    auth.username = username;
> > -    if (auth.username)
> > -      return ui.input('Enter your GitHub password or access token', null, true);
> > -  })
> > -  .then(function(password) {
> > -    auth.password = password;
> > -    if (!auth.username)
> > -      return false;
> > -
> > -    return ui.confirm('Would you like to test these credentials?', true);
>
> Left this as-is (with the old code removed) since we don't need to allow
> this old stuff to go in.
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/jspm/github/pull/90/files/7e73f5e239dac1f8eca852495dc8029f82991f26#r66714112>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAkiyrxzLDoQA_BvR-jvnJFmFE3VFUQcks5qK1eZgaJpZM4Izqxv>
> .
>